packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -51,27 +51,16 @@ class DaemonCommand extends FlutterCommand {
   Future<FlutterCommandResult> runCommand() async {
     globals.printStatus('Starting device daemon...');
     isRunningFromDaemon = true;
-
-    final NotifyingLogger notifyingLogger = NotifyingLogger();
-
     Cache.releaseLockEarly();
 
-    await context.run<void>(
-      body: () async {
-        final Daemon daemon = Daemon(
-          stdinCommandStream, stdoutCommandResponse,
-          notifyingLogger: notifyingLogger,
-        );
-
-        final int code = await daemon.onExit;
-        if (code != 0) {
-          throwToolExit('Daemon exited with non-zero exit code: $code', exitCode: code);
-        }
-      },
-      overrides: <Type, Generator>{
-        Logger: () => notifyingLogger,
-      },
+    final Daemon daemon = Daemon(
```

Comment:
Will we be able to get verbose logging from the daemon anymore (I was literally using that to debug something yesterday)?

Reply:
I'll update the top level to check for daemon + verbose so that still works

Reply:
actually, I confused myself here: how were you using the verbose logging? I guess the way it works today is that you get verbose logging up until we hit the runZoned in daemon right?

Reply:
```
$ flutter daemon --verbose
[  +10 ms] executing: [/Users/m/Projects/flutter/] git -c    
log.showSignature=false log -n 1 --pretty=format:%H
[  +40 ms] Exit code 0 from: git -c log.showSignature=false log -n 1
--pretty=format:%H
... blah blah verbose
[   +5 ms] Starting device daemon...
```
stdin:
```
[{"event":"daemon.connected","params":{"version":"0.5.3","pid":35828}}]
[{"id":1,"method":"device.enable"}]
[+2399 ms] executing: /usr/bin/xcode-select --print-path
[   +7 ms] Exit code 0 from: /usr/bin/xcode-select --print-path
[        ] /Users/m/Applications/Xcode-11-5-beta2.app/Contents/Developer
[   +1 ms] executing: /usr/bin/xcodebuild -version
[ +111 ms] Exit code 0 from: /usr/bin/xcodebuild -version
... blah blah still verbose
```

Reply:
WDUT about the verbose messages from the notifying logger?

```
[{"event":"daemon.connected","params":{"version":"0.5.3","pid":15152}}]
[{"event":"daemon.logMessage","params":{"level":"trace","message":"executing: [C:\\Users\\Jonah\\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H"}}]
[{"event":"daemon.logMessage","params":{"level":"trace","message":"Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H"}}]
[{"event":"daemon.logMessage","params":{"level":"trace","message":"34736b7568880662af9cf42bdc383154f7ed07f1"}}]
```

Reply:
I'm fine with that. Or just not having verbose available if it's too painful.